recipes/monotonicnetworks/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
LICENSE is in the repository. If you use python [build](https://github.com/pypa/build) to create the sdist and wheel, it should include the LICENSE automatically. The way to do it before was to create a `MANIFEST.in` file but that shouldn't be needed anymore.

Reply:
Thanks! Good to know.